server/src/lib/model/db/migrations/20191029113543-change-visible-use-clients.ts

```diff
@@ -0,0 +1,11 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    ALTER TABLE user_clients MODIFY visible TINYINT(2) DEFAULT 0;
```

Comment:
This seems okay to me, since `visible` already expects a type of `0 | 1`. Note that you'll need to add `| 2` to `/common/user-clients.ts`.

Can you write a bit more about what each value will represent?

Reply:
Ah, nevermind. I just saw it's updated in https://github.com/mozilla/voice-web/pull/2256/.

Reply:
A little comments would be good though, I will add that.